### PR TITLE
Add 2020 Corolla Hybrid support

### PR DESCRIPTION
**Pull request by hdoublearp**
Added fingerprint for 2020 Corolla Hybrid LE. Used DBC from CHRH and lateral tuning values from Corolla Hatch.

Please sanity check and review. It was unclear to me whether or not an entry should be added to NO_DSU_CAR, but I figured TSSP2_CAR and NO_STOP_TIMER_CAR should suffice. Please correct me if I'm wrong.

Best,
Christian

### CODE REVIEW

**Comment by rbiasini** [comment]
Nice. Correct, no need to add the car to NO_DSU_CAR.

Can you please post here (or DM me on Discord) a route taken with stock mode and one with OP working?
thanks!

**Comment by thesoundofom** [comment]
Sure, I'll take care of that tomorrow morning (coffee run). However, according to the CI pipeline, my indents need to be fixed... feel free to edit that,

File: selfdrive/car/toyota/interface.py [hunk 1]
```diff
@@ -179,6 +179,16 @@ def get_params(candidate, fingerprint, vin=""):
       ret.mass = 3060. * CV.LB_TO_KG + std_cargo
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.3], [0.05]]
       ret.lateralTuning.pid.kf = 0.00007818594
+	  
+	elif candidate == CAR.COROLLAH:
+      stop_and_go = True
```
Comment (rbiasini) [inline selfdrive/car/toyota/interface.py]: bad indent

Comment (thesoundofom) [reply selfdrive/car/toyota/interface.py]: My bad. I just pushed fixed indent.

**Comment by thesoundofom** [comment]
Closing pull request pending further testing with proper denso radar DBC. Thanks to @wocsor for finding the issue! Will resubmit later.

**Pull request closed without merging**